README.md

```diff
@@ -185,13 +185,39 @@ Demonstrates [module per config](#module-per-config).
 
 # Specials
 
-To run the webapp, example and test modules of 
-https://github.com/frankframework/frankframework you can use the build.xml
-files in the specials folder. As with other projects the iaf folder needs to be
-a sibling folder of the frank-runner folder. See also the
-build-example.properties files in these folders that describe several
-properties that can be set to speed up and customize the build.
-
+The main purpose of the Frank!Runner is to run released code of the
+Frank!Framework - you do that by running `restart.bat` or `restart.sh` in the
+root directory of the Frank!Runner. When you are doing development work on
```

Comment:
It is more common to run the restart.bat, restart.sh (or in Eclipse the build.xml) in your project folder. Maybe it's better to link to a different section of this README instead of explaining this.

Reply:
I leave the explanation out entirely. It is quite obvious where to find this information. My intention was to write an introduction to this section, not to explain something already explained.